package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "language-miniyaml",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
```suggestion
  "version": "0.0.2",
```

I think version 1.0.0 is exaggerated. This was based on [yaml version 0.3.2](https://github.com/atom/language-yaml/blob/master/package.json#L3) and all the other https://github.com/github/linguist/blob/master/.gitmodules stay below `0.0.X` or `0.X.0` respectively.